source/release-notes.txt

```diff
@@ -11,6 +11,36 @@ Release Notes
    :class: twocols
 
 
+|compass| 1.43.2
+----------------
+
+*Released June 25, 2024*
+
+New Features:
+
+- Shows tool tip when a query or aggregation is generated without
+  content (:issue:`COMPASS-7837`).
+- Confirmation dialog when quitting Compass (:issue:`COMPASS-6435`).
```

Comment:
(parallelism)
Can you start each list item with a verb here?
Adds a confirmation dialog ... etc.

Reply:
Done.